script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -133,34 +133,34 @@ The upperclassman is staring at me.
 Not only that, but she comes sauntering up to me.
 <Page36>
 "Good morning. I'm sorry for bothering you so early in the day."
-"Ah, no, I'm sorry too. For staring at you, that is."
+"Ah, no, I'm sorry too. For staring at you, I mean."
 The smile on her face made me apologize by reflex.
 <Page37>
 "No, it's my fault for doing something conspicuous.
 I accidentally let you see an embarrassing side of your upperclassman.
 But it's true what they say about not being good in the morning, right?"
 <Page38>
 Senpai chuckles.
-I am taken aback by her gesture.
+I'm taken aback by her gesture.
 <Page39>
 "Look. It's me, it's me. I met you just last week, have you forgotten about me already?"
 <Page40>
-"Huh, ...Ciel-senpai...?"
+"Huh...  Ciel-senpai...?"
 In an instant, my memory is jolted.
 Of course, she looks familiar. I've met this person several times at school.
 Actually, there's probably no one in our school who doesn't know Ciel-senpai.
 <Page41>
 At some point, she started to get called 'The Almighty Godfather Senpai'.
 If there are first years who are troubled, she will counsel and help them feel better.
 If there are second years who are lost, she will guide them herself.
-If there are students in trouble with the third years, she will stop them from bullying their underclassmen.
+If there are third years making trouble with other students, she will stop them from bullying their underclassmen.
 <Page42>
-Some students even call her the real student council president, because she is more dependable than the teachers and the school council themselves.
+Some students even call her the real student council president, since she's more dependable than the teachers and the school council themselves.
 It goes without saying that when I ran into trouble last week, she helped me out.
 <Page43>
 ...It's actually a bit of a blur.
 I'm not sure if it's because I'm not a morning person, or because my head got foggy from the thought of going back to that home from seven years ago.
 I...
-C:>Sadly, there's nothing to talk to her about.
-C:>I have a few questions.
-C:>Since we're here, I should just carefully admire her.
+C:>have nothing to talk to her about.
+C:>have a few questions.
+C:>should just carefully admire her, since I have the opportunity.
```

Comment:
Have you playtested that this fits? IIRC choices are a bit more limited in line-length